pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -230,13 +231,115 @@ func (adc *attachDetachController) Run(stopCh <-chan struct{}) {
 	// 	return
 	// }
 
+	err := adc.populateActualStateOfWorld()
+	if err != nil {
+		glog.Errorf("Error populating the actual state of world: %v", err)
```

Comment:
Use `runtime.HandleError` instead of `glog.Errorf` everywhere.

Errors need to start with a lowercase letter.

e.g. `runtime.HandleError(fmt.Errorf("error populating the actual state of world: %v", err))`